assets/js/components/ClusterDetails/BackToCluster.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Button from '@components/Button';
+import { useNavigate } from 'react-router-dom';
+import { EOS_ARROW_BACK } from 'eos-icons-react';
+
+export const BackToCluster = ({ clusterId }) => {
```

Comment:
maybe we could generalize that, by creating a BackButton component and passing a prop/child

Reply:
Maybe just asking the `url` and `text`, without making it more complex.

Reply:
Yeah, @arbulu89 bear in mind that you can also use `children` for that and just pass the text as `<BackLink>Back to cluster details</BackLink>`